torchtune/modules/kv_cache.py

```diff
@@ -53,7 +51,7 @@ def reset(self) -> None:
 
     @property
     def size(self) -> int:
-        return self.cache_pos[0].item()
+        return int(self.cache_pos[0].item())
```

Comment:
how come this was added?

Reply:
It's a leftover from a version that would raise a type error. I'll remove it, thanks!